src/createXcmCalls/polkadotXcm/limitedTeleportAssets.ts

```diff
@@ -31,7 +31,9 @@ export const limitedTeleportAssets = (
 	xcmVersion: number,
 	specName: string,
 	registry: Registry,
-	weightLimit?: string,
+	isLimited?: boolean,
```

Comment:
I think we should move away from adding a bunch of optional params at the end of the args, and instead put them in a single option.